traits/has_traits.py

```diff
@@ -630,17 +626,12 @@ def update_traits_class_dict(class_name, bases, class_dict):
 
             # Add the view element to the class's 'ViewElements' if it is
             # not already defined (duplicate definitions are errors):
-            if name in view_elements.content:
+            if name in view_elements:
```

Comment:
This shouldn't be possible any more: the names are simply keys of the class dictionary. But I guess that may mean that we still need the duplication check elsewhere.

Reply:
I was wondering about that.  I don't think it's possible.  In the original code `ViewElements` doesn't have the parent classes attached yet, and `content` is a simple dictionary in any case, so I don't know why that code was in there.  I will remove it.